### PR TITLE
Deleting cache files

### PR DESCRIPTION
**Pull request by amirdedeic**
removing and ignoring the .idea folder

### CODE REVIEW

File: project_backend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# JetBrains trash
+.idea/
```
Comment (nacongo) [inline project_backend/.gitignore:2]: You should add here `pycache` folders as well:
project_backend/\__pycache__
tutorials/\__pycache__

**Review by nacongo — CHANGES_REQUESTED** [review]
Please add these changes to the new branch and create new pull request.
Currently you've added these changes on existing `deleting_cache_files` branch and github recognized changes that we've already merged to master.
You can call new branch something like `add_gitignore_file`.

And also make sure to add appropriate title and the description of the pull request as well.

File: project_backend/.gitignore (hunk 1 above)
Comment (amirdedeic) [reply project_backend/.gitignore:2]: Ok. Should I close this pull request?

Comment (nacongo) [reply project_backend/.gitignore:2]: Yes. You can just close this one.

**Pull request closed without merging**